contracts/bondingcurve/CommonsToken.sol

```diff
@@ -147,6 +148,12 @@ contract CommonsToken is BondingCurveToken, Pausable {
     minExternalContribution = _settings[7];
 
     externalToken = ERC20(_addresses[0]);
+
+    initialRaiseInternal = _calcInternalTokens(initialRaise);
+  }
+
+  function _calcInternalTokens(uint256 externalTokens) internal view returns (uint256) {
```

Comment:
I would suggest moving this to the bottom of the file as by convention the public methods are first, at the top of the file.

Reply:
Sounds ok.